tests/shared/src/test/scala/scala/meta/tests/parsers/BasePositionSuite.scala

```diff
@@ -24,30 +24,29 @@ abstract class BasePositionSuite(defaultDialect: Dialect) extends ParseSuite {
    *   checkPositions[Type](
    *     "[X] =>> (X, X)",
    *     """|Type.Bounds [X@@] =>> (X, X)
-   *        |Type.Tuple (X, X)
-   *        |""".stripMargin
+   *         |Type.Tuple (X, X)
+   *         |""".stripMargin
    *   )
    * }}}
    *
-   * Every line in the output format shows the AST tree node type and the
-   * tokens for that tree node. Offset positions are rendered as "@@".
+   * Every line in the output format shows the AST tree node type and the tokens for that tree node.
+   * Offset positions are rendered as "@@".
    *
    * Below is an example bug that is easy to catch with position tests.
    *
    * {{{
    *   checkPositions[Stat](
    *     "trait A { self: B => }",
    *     """|Ctor.Primary trait A @@{ self: B => }
-   *        |Name.Anonymous {
-   *        |Template { self: B => }
-   *        |Self self: B
-   *        |""".stripMargin
+   *         |Name.Anonymous {
+   *         |Template { self: B => }
+   *         |Self self: B
+   *         |""".stripMargin
```

Comment:
There shouldn't be an additional space.
I'm looking how to fix that - seems like an combination of parser and fmt issues.

Reply:
see comment just above